content/en/apps/reference/app-settings/_index.md

```diff
@@ -224,6 +224,7 @@ matches "mary", "Mary", "john", "John", "JOhN", etc. Not "maryjohn"
 |----|----|
 | `unique(*fields)` | Returns `true` if no existing valid report has the same value for all of the listed fields. Fields are compared to those at the top level and within `fields` for every report doc. To include the form type use `'form'` as one of the fields. Eg `unique('form', 'patient_id')` checks that this form was never submitted for this patient. |
 | `uniquePhone(field)` | Returns `true` if contact with the phone number already exists. _Added in 4.3.0_ |
+| `validPhone(field)` | Returns `true` if the field is a valid phone number else returns false. _Added in 4.3_ |
```

Comment:
This isn't available in 4.3.0, right? Let's be specific about the version. Not sure whether this will be a 4.3.1 or 4.4.0 but leaving this as a reminder for once the decision has been made.

```suggestion
| `validPhone(field)` | Returns `true` if the field is a valid phone number else returns false. _Added in 4.3.1_ |
```

Reply:
@yrimal @PrjShrestha Please review this suggestion.